### DISCUSSION TITLE
ScriptReference/AssetDatabase.AddObjectToAsset

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#253] [Notes]
# ScriptReference/AssetDatabase.AddObjectToAsset

https://docs.unity3d.com/ScriptReference/AssetDatabase.AddObjectToAsset.html

**Comment by radiatoryang** [comment]
To access sub-assets later, see [`AssetDatabase.LoadAllAssetRepresentationsAtPath`](https://docs.unity3d.com/ScriptReference/AssetDatabase.LoadAllAssetRepresentationsAtPath.html).  
To get the main asset path, use [`AssetDatabase.GetAssetPath`](https://docs.unity3d.com/ScriptReference/AssetDatabase.GetAssetPath.html).
